p2p/src/test/kotlin/maru/p2p/EnrStringUtil.kt

```diff
@@ -25,7 +25,7 @@ fun getBootnodeEnrString(
   val bootnodeNR =
     NodeRecordBuilder()
       .nodeRecordFactory(NodeRecordFactory(IdentitySchemaInterpreter.V4))
-      .seq(1)
```

Comment:
Can you please explain this change?

Reply:
in discovery v5 there is a mechanism to update the node. The ping message contains the sequence number. If the seq number is higher than the one in the buckets the latest version is requested.

Reply:
What is a node update?

Reply:
We don't have ping message. Should the current seq number be added to the status message since we are using that like a ping message?